src/devices/ikea.ts

```diff
@@ -1138,6 +1138,13 @@ const definitions: Definition[] = [
         description: 'STOFTMOLN ceiling/wall lamp 15 warm light dimmable',
         extend: [tradfriLight()],
     },
+    {
+        zigbeeModel: ['JETSTROM 3030 ceiling'],
+        model: 'L2205, L2206 JETSTRÖM',
```

Comment:
```suggestion
        model: 'L2205/L2206',
```

Can we use this here?

Reply:
I got the model from the manual (https://www.ikea.com/gb/en/manuals/jetstroem-led-wall-light-panel-smart-dimmable-wired-in-colour-and-white-spectrum__AA-2416878-1-100.pdf) but thinking about it, it is only one product so not sure where the other model comes into it.   Looking at the back it of the device it says L2206 so just going with that